scip-java/src/main/scala/com/sourcegraph/scip_java/buildtools/BuildTool.scala

```diff
@@ -20,10 +20,28 @@ abstract class BuildTool(val name: String, index: IndexCommand) {
 object BuildTool {
   def all(index: IndexCommand): List[BuildTool] =
     List(
+      new ScipBuildTool(index),
       new BazelBuildTool(index),
       new GradleBuildTool(index),
       new MavenBuildTool(index),
+      new SbtBuildTool(index),
+      new MillBuildTool(index)
+    )
+
+  def autoOrdered(index: IndexCommand): List[BuildTool] =
```

Comment:
Is there a reason we can't make the `all()` method return popularity-ordered build tools?

Reply:
Bazel being a supported (but not supported?) auto tool is one reason - but I can just implement one method in terms of the other if you think it'll be clearer

Reply:
done in https://github.com/sourcegraph/scip-java/pull/582/commits/c20606f889c5a5f5a60a91fd5feb0a1064bc4086